packages/byods/package.json

```diff
@@ -25,10 +25,12 @@
     "docs": "typedoc --emit none",
     "fix:lint": "eslint 'src/**/*.ts' --fix",
     "fix:prettier": "prettier \"src/**/*.ts\" --write",
-    "prebuild": "rimraf dist"
+    "prebuild": "rimraf dist",
+    "test": "yarn test:style && yarn test:unit",
+    "test:style": "eslint 'src/**/*.ts'",
+    "test:unit": "jest --config=jest.config.js --runInBand"
   },
   "devDependencies": {
-    "@babel/preset-typescript": "7.16.7",
```

Comment:
Why was this removed?

Reply:
When we use Babel, we need this package to handle typescript files but I am using ts-jest to handle the TypeScript transpilation so this package doesn't seem to be required